src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,32 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: (string | undefined)[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null | undefined) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location !== undefined && places.indexOf(location)) {
+      places.push(location);
+    }
+    return {
+      '@type': 'Person',
+      name: authorName,
+      ...(sameAs.length && { sameAs }),
+      ...(authorImage && { image: authorImage }),
+    };
+  };
+
+  const bylineAuthors =
+    bylineLinkedData?.length !== undefined && bylineLinkedData?.length > 1
```

Comment:
Not sure we need this
```suggestion
   bylineLinkedData?.length > 1
```

Reply:
me neither, but without it you get the typescript error `'bylineLinkedData.length' is possibly 'undefined'.`, wonder if theres a cleverer way to fix this or if im missing something, but this seemed the simplest (if a bit wordy) fix

Reply:
ah actually i can add the undefined to the type definition of BylineLinkedData instead, it needs it as it's an optional parameter - this means i can commit some of your suggestions below too 🎉

Reply:
actually, i can wrap it in an if statement checking if(bylineLinkedData) and then it will work - bit cleaner and will allow for some of your suggestions below too?